README.rst

```diff
@@ -24,10 +24,8 @@ Code Overview
 
 The program ``pipelines.py`` creates a graph using the Python package ``networkx``, and then uses the Ocean software tools to run the ``minimum_vertex_cover`` function from within the ``dwave_networkx`` package.
 
-Inside the Code: The Lagrange Parameter
------- --- ----  --- -------- ---------
-Gamma, our Lagrange parameter, weights the constraints in the problem versus
-the energy term (objective). If the Lagrange parameter is too small, relative
+Gamma, our Lagrange parameter (please see <https://en.wikipedia.org/wiki/Lagrange_multiplier> for a discussion of Lagrange multiplier, an analogous concept), weights the constraints in the problem versus
```

Comment:
![image](https://user-images.githubusercontent.com/34041130/70743639-0a9e9c00-1cd5-11ea-9e41-8c9bce2f1842.png)

"see <https://en.wikipedia.org/wiki/Lagrange_multiplier> for" --> change to "see `Lagrange Multiplier <https://en.wikipedia.org/wiki/Lagrange_multiplier>`_ for" to get a nicer display

Reply:
@JoelPasvolsky, thanks. I fixed this.